templates/vhost/_passenger.erb

```diff
@@ -25,11 +25,14 @@
 <% if @passenger_user -%>
   PassengerUser <%= @passenger_user %>
 <% end -%>
+<% if @passenger_group -%>
+  PassengerGroup <%= @passenger_group %>
+<% end -%>
 <% if @passenger_high_performance -%>
   PassengerHighPerformance <%= scope.call_function('apache::bool2httpd', [@passenger_high_performance]) %>
 <% end -%>
 <% if @passenger_nodejs -%>
-  PassengerNodejs <%= @passenger_nodejs -%>
+  PassengerNodejs <%= @passenger_nodejs %>
```

Comment:
nice catch! i hope the config parser is hardy enough to handle a newline :)

Reply:
I don't think so 😉 

```
PassengerNodejs takes one argument, The Node.js command to use.
```

Reply:
ah ok, so we def need that newline